presidio-analyzer/analyzer/predefined_recognizers/spacy_recognizer.py

```diff
@@ -15,6 +15,12 @@ def load(self):
         # preprocessed nlp artifacts
         pass
 
+    @staticmethod
+    def build_spacy_explanation(recognizer_name, original_score):
+        explanation = AnalysisExplanation(recognizer_name, '',
```

Comment:
Here we could provide an example of how to add a textual explanation

Reply:
Please add argument names to function call, and some text about this being identified by Spacy as the interpretation